src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -16,7 +16,7 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
   <ItemGroup>
     <TargetingPackExclusions Include="System.Linq.Expressions" />
-    <ProjectReference Include="..\src\System.Linq.Expressions.csproj" />
+    <RuntimeReferencedAssemblyNames Include="System.Linq.Expressions" />
```

Comment:
The tests use some types which are public in the source project, but don't exist in the reference assembly. It's essentially `InternalsVisibleTo`.